build/Targets/GenerateInternalsVisibleTo.targets

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```

Comment:
Wow, thanks. I didn't notice this before, and I certainly didn't add this on purpose. I'll remove it.